snyk-monitor/README.md

```diff
@@ -204,7 +204,7 @@ By default, `snyk-monitor` does not scan containers that are internal to Kuberne
 
 If you prefer to override this, you can add your own list of namespaces to exclude by setting the `excludedNamespaces` to your own list. For example:
 ```yaml
---set excludedNamespaces={kube-node-lease,kube-public,local-path-storage,some_namespace}
+--set excludedNamespaces="{kube-node-lease,kube-public,local-path-storage,some_namespace}"
```

Comment:
I thought the Helm template would handle this - https://github.com/snyk/kubernetes-monitor/blob/573dd2a98b5d6fb8b950810125c89f56f2caf4ae/snyk-monitor/templates/configmap.yaml#L13-L15

Does it throw an error now?

Reply:
I get this:


```Error: "helm upgrade" requires 2 arguments

Usage:  helm upgrade [RELEASE] [CHART] [flags]```


which disappears as soon as put the quotes in 🤷